src/features/expense/ui/SelectMonthList/SelectMonthList.tsx

```
@@ -0,0 +1,51 @@
+    onSetPeriod(newPeriod);
+  };
+
+  return (
+    <ul className='list-none flex flex-col gap-6 max-h-60 overflow-y-auto'>
+      {periods.reverse().map(({ year, month }) => (
+        <SelectMonthListItem
+          year={year}
+          month={month}
+          selected={period.year === year && period.month === month}
+          handleClick={handleClickMonth}
+        />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)